### PR TITLE
Remove C from sympy.solvers for non-cylic imports

### PR DESCRIPTION
**Pull request by aktech**
Remove C from sympy.solvers for non-cylic imports.

### CODE REVIEW

**Comment by aktech** [comment]
ping @toolforger

**Comment by toolforger** [comment]
+1 provided Travis is okay (modulo pypy woes)

File: sympy/solvers/solveset.py [hunk 1]
```diff
@@ -278,17 +280,19 @@ def _is_function_class_equation(func_class, f, symbol):
     ========
 
     >>> from sympy.solvers.solveset import _is_function_class_equation
-    >>> from sympy import C, tan, sin, tanh, sinh, exp
```
Comment (toolforger) [inline sympy/solvers/solveset.py]: Good catch. C should never be necessary in a docstring.
Did you run the doctests? (Don't worry if not - Travis will do doctesting anyway.)

Comment (aktech) [inline sympy/solvers/solveset.py]: Yes, I did run the doctests.
It was good.

**Pull request merged**